---
title: '217 基于单细胞甲基化的小鼠脑细胞图谱'
date: '2022-02-23'
archive: ["2022","2022-02","2022-02-23"]
categories:
  - 学术报告
tags: [talks, DNA methylation, single-cell epigenomics, neuroscience, mouse brain atlas, cell-type-specific enhancer]
show_comments: true
thumbnail: ""
---
- 题目：217-基于单细胞甲基化的小鼠脑细胞图谱
- 地点：- Zoom会议 ID：861 0146 4725 密码：269044
- Zoom会议链接：https://us06web.zoom.us/j/86101464725?pwd=bXphV2NYdGRZeVRaZys0WnNjczF4Zz09
- 主讲人刘翰青，男，本科毕业于浙江大学竺可桢学院，目前为加州大学圣地亚哥分校生物博士（在读），导师为索尔克研究所 Joseph Ecker 教授。主要研究方向为分子神经生物学和单细胞表观基因组学。博士期间参与数项 BRAIN Initiative 课题，研究和开发单细胞表观遗传组学技术及其分析方法，并以此构建小鼠脑细胞图谱，理解大脑细胞类型多样性的分子生物学基础。多项研究成果近日发表于 Nature 脑计划专刊。

<div align="center">
<img src="https://github.com/cgmonline/cgmonline/blob/master/image/2022_Hanqing_Liu.jpg?raw=true" height=250>
</div>

# 中文摘要
Mammalian brain cells show remarkable diversity in gene expression, anatomy and function, yet the regulatory DNA landscape underlying this extensive heterogeneity is poorly understood. Here we carry out a comprehensive assessment of the epigenomes of mouse brain cell types by applying single-nucleus DNA methylation sequencing1,2 to profile 103,982 nuclei (including 95,815 neurons and 8,167 non-neuronal cells) from 45 regions of the mouse cortex, hippocampus, striatum, pallidum and olfactory areas. We identified 161 cell clusters with distinct spatial locations and projection targets. We constructed taxonomies of these epigenetic types, annotated with signature genes, regulatory elements and transcription factors. These features indicate the potential regulatory landscape supporting the assignment of putative cell types and reveal repetitive usage of regulators in excitatory and inhibitory cells for determining subtypes. The DNA methylation landscape of excitatory neurons in the cortex and hippocampus varied continuously along spatial gradients. Using this deep dataset, we constructed an artificial neural network model that precisely predicts single neuron cell-type identity and brain area spatial location. Integration of high-resolution DNA methylomes with single-nucleus chromatin accessibility data3 enabled prediction of high-confidence enhancer–gene interactions for all identified cell types, which were subsequently validated by cell-type-specific chromatin conformation capture experiments4. By combining multi-omic datasets (DNA methylation, chromatin contacts, and open chromatin) from single nuclei and annotating the regulatory genome of hundreds of cell types in the mouse brain, our DNA methylation atlas establishes the epigenetic basis for neuronal diversity and spatial organization throughout the mouse cerebrum.

# 参考文献
Liu, Hanqing, Jingtian Zhou, Wei Tian, Chongyuan Luo, Anna Bartlett, Andrew Aldridge, Jacinta Lucero, et al. 2021. “DNA Methylation Atlas of the Mouse Brain at Single-Cell Resolution.” Nature 598 (7879): 120–28.

# 关键词
DNA methylation, single-cell epigenomics, neuroscience, mouse brain atlas, cell-type-specific enhancer
